apps/rush-lib/src/api/RushProjectConfiguration.ts

```diff
@@ -253,10 +239,48 @@ export class RushProjectConfiguration {
     }
   }
 
+  /**
+   * Load only the `incrementalBuildIgnoredGlobs` property from the rush-project.json file, skipping
+   * validation of other parts of the config file.
+   *
+   * @remarks
+   * This function exists to allow the ProjectChangeAnalyzer to load just the ignore globs without
+   * having to validate the rest of the `rush-project.json` file against the repo's command-line configuration.
```

Comment:
If both functions get called, will the file be loaded twice?

Reply:
No. `heft-config-file` has a cache.